.github/workflows/build-releases.yml

```
@@ -0,0 +1,260 @@
+          3. Move the binary to your PATH
+          4. Run `gitai --help` to verify installation
+          
+          ## Checksums
+          
+          EOF
+          
+          # Generate checksums
+          find artifacts -type f \( -name "*.tar.gz" -o -name "*.zip" \) -exec sha256sum {} \; >> RELEASE_NOTES.md
+          
+      - name: Create GitHub Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)